src/module.c

```diff
@@ -123,6 +123,16 @@ bool ACLUserMayAccessIndex(RedisModuleCtx *ctx, IndexSpec *sp) {
   return ret;
 }
 
+// Validates ACL key-space permissions w.r.t the given index spec for Redis
+// Enterprise environments only.
+static inline bool checkEnterpriseACL(RedisModuleCtx *ctx, IndexSpec *sp) {
+  if (!IsEnterprise() || ACLUserMayAccessIndex(ctx, sp)) {
+    // Either OSS, or ACL check passed.
+    return true;
+  }
+  return false;
```

Comment:
Not critical but this can be changed to: `return !IsEnterprise() || ACLUserMayAccessIndex(ctx, sp)` no?

Reply:
Sure, changing